### PR TITLE
Use CSS `user-select` to achieve easy copy paste

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [x] Link to relevant GitHub issues or pull requests

- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

Pressing Ctrl+A selects everything inside code. I tried some fancier CSS selectors but the better solution is just marking that which should not be selected.

removes #852 and uses CSS instead

### CODE REVIEW

File: plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css [hunk 1]
```diff
@@ -1,5 +1,23 @@
+/* Custom Jenkins bits to provide a good user experience
```
Comment (timja) [inline plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css]: could this go in a different css file? this file is completely generated currently from the prism website

Comment (casz) [reply plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css]: Sure not a problem :)

File: plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+iframe,
+.yui-overlay,
+#page-head,
+#menuSelector,
+footer
+ {
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
```
Comment (timja) [inline plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css]: looks fine, shame about the prefixing:
https://caniuse.com/#feat=user-select-none

Comment (casz) [reply plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css]: The docs is a mess :cry: 
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

**Pull request merged**